src/workloads/execution/SecondaryReadsGenny.yml

```diff
@@ -0,0 +1,252 @@
+SchemaVersion: 2018-07-01
+Owner: "@TODO" # what should go here? us or the owners of secondary_reads? 
```

Comment:
Let's add someone from replication to this PR. I think it would be great of replication team owned this. 

@ldennis @lankas could you recommend someone from your teams to review this workload? As described, Simon is porting over the legacy workload of the same name.